tripal_chado/src/Plugin/Field/FieldFormatter/ChadoAnalysisFormatterDefault.php

```diff
@@ -73,23 +73,8 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       $list[$delta] = $renderable_item;
     }
 
-    // If only one element has been found, don't make into a list.
-    if (count($list) == 1) {
-      $elements = $list;
-    }
-
-    // If more than one value has been found, display all values in an
-    // unordered list.
-// @todo: add a pager
-    elseif (count($list) > 1) {
-      $elements[0] = [
-        '#theme' => 'item_list',
-        '#list_type' => 'ul',
-        '#items' => $list,
-        '#wrapper_attributes' => ['class' => 'container'],
-      ];
-    }
-
+    // If multiple items, convert to a list.
+    $elements = $this->createListMarkup($list);
```

Comment:
I confirmed that `createListMarkup()` does not create a list out of singular items but this comment is still misleading.


```suggestion
    // Will convert $list to a markup list if there are > 1 items.
    $elements = $this->createListMarkup($list);
```

Reply:
Resolved with commit https://github.com/tripal/tripal/pull/2203/commits/fb7fefe774e0ab2c1f968bf995dbe565a8956865